ansible/inventory/yandex_cloud.yml

```diff
@@ -0,0 +1,13 @@
+vms:
+  hosts:
+    # I won't give you the IPs :troll:
```

Comment:
anyway, it should dynamic if u use dynamic inventory. Also, there can be multiple machines

Reply:
> anyway, it should dynamic if u use dynamic inventory. Also, there can be multiple machines

Man you literally are commenting on a static inventory file while there is dynamic one in the same folder